src/Views/Homepage.vala

```diff
@@ -428,6 +439,18 @@ public class AppCenter.Homepage : Adw.NavigationPage {
         });
     }
 
+    private int get_app_scale_factor () {
```

Comment:
every Gtk.Widget has access to `scale_factor`: https://valadoc.org/gtk4/Gtk.Widget.scale_factor.html

So this function should be unnecessary